### PR TITLE
initial info file added which turns off nearly all default drupal CSS…

### PR DESCRIPTION
**Pull request by AlexBrown-appnovation**
- removed core CSS files (will add them back in where necessary)
- removed core module CSS files (will add them back in where necessary)
- added design standards CSS files
- added design standards JavaScript files
- created default regions (totally placeholder at the moment)

### CODE REVIEW

File: us_web_design_standards.info [hunk 1]
```diff
@@ -1 +1,66 @@
-name = U.S. Web Design Standards
+name        = US Web Design Standards
+description = Open source UI components and visual style guide to create consistency and beautiful user experiences across U.S. federal government websites.
+screenshot  = screenshot.png
```
Comment (fureigh) [inline us_web_design_standards.info]: Feel free to borrow the placeholder `screenshot.png` from the 8.x branch if you'd like.

File: us_web_design_standards.info [hunk 2]
```diff
@@ -1 +1,66 @@
...
+stylesheets[all][] = tracker.css
+stylesheets[all][] = update.css
+stylesheets[all][] = user.css
+
+;; add our own CSS files from the standards folder
+stylesheets[all][] = assets/css/google-fonts.css
+stylesheets[all][] = assets/css/main.css
+stylesheets[all][] = assets/css/overrides.css
+
+;; we may need to remove jquery here and recommend jquery_update module to users
```
Comment (fureigh) [inline us_web_design_standards.info:51]: Thanks for noting this. Perhaps share findings in https://github.com/18F/web-design-standards-drupal/issues/2?

File: us_web_design_standards.info [hunk 3]
```diff
@@ -1 +1,66 @@
-name = U.S. Web Design Standards
+name        = US Web Design Standards
```
Comment (fureigh) [inline us_web_design_standards.info]: Let's change 'US' to 'U.S.' for consistency with https://playbook.cio.gov/designstandards/ et al.

**Comment by fureigh** [comment]
@AlexBrown-appnovation Thanks for this! I made some inline comments. The two main changes I'd suggest are:
1. Change 'US' to 'U.S.' and
2. Change spacing to be compliant with [Drupal coding standards](https://www.drupal.org/coding-standards#operators), though I do appreciate the readability of the tab-based approach you used.

**Comment by AlexBrown-appnovation** [comment]
- 'US' has been changed to 'U.S.'
- spacing around operators is now up to Drupal Standards

File: us_web_design_standards.info [hunk 4]
```diff
@@ -1 +1,66 @@
 name = U.S. Web Design Standards
+description = Open source UI components and visual style guide to create consistency and beautiful user experiences across U.S. federal government websites.
+screenshot = screenshot.png
+core = 7.x
+engine = phptemplate
+
+;;  remove some of the system (core) CSS files
```
Comment (fureigh) [inline us_web_design_standards.info:7]: I'll merge this, but: why two semicolons instead of one?

**Comment by fureigh** [comment]
@AlexBrown-appnovation :+1: Thanks! Curious to hear about the double semicolons but merging for now.

**Comment by AlexBrown-appnovation** [comment]
The double semi colons are just my commenting style, if its just one line being commented out I use one, but if I'm writing a block of text and keeping it within 80 characters in width I'll use two.

**Comment by fureigh** [comment]
Got it. For the sake of readability, where the Drupal coding standards exist, I'd ask that we generally stay in line with them.

**Pull request merged**